src/app/users/view-user/view-user.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="container m-b-20" fxLayout="row" fxLayout.lt-md="column" fxLayoutAlign="end" fxLayoutGap="2%">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+  <button mat-raised-button color="warn">
+    <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+    Delete
+  </button>
+  <button mat-raised-button color="primary">
+    <fa-icon icon="cog"></fa-icon>&nbsp;&nbsp;
+    Change Password
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="header">
```

Comment:
Use Angular Material Typography instead of SCSS for header as mentioned in the task description. You can refer more recently created components like: https://github.com/openMF/web-app/blob/master/src/app/system/account-number-preferences/view-account-number-preference/view-account-number-preference.component.html